client/app/(auth)/createProfile.tsx

```diff
@@ -1,19 +1,430 @@
+              <View className="my-8 size-8 self-end" />
+
+              <Text className="pbk-h5 mb-8 text-base-white">
+                Tell us who you are
+              </Text>
+
+              <Text className="pbk-b2 mb-1.5 text-base-white">Username</Text>
+              <Controller
+                control={control}
+                name="username"
+                render={({ field: { onChange, value } }) => (
+                  <View
+                    className={`mb-2 min-h-14 w-full flex-row items-center rounded-xl border ${errors.username ? "border-red-600" : "border-gray-920"} px-2 py-2`}
+                  >
+                    <TextInput
+                      autoCapitalize="none"
+                      className="flex-1 text-base-white placeholder:pbk-b1"
+                      onChangeText={(text) => {
+                        onChange(text);
+                      }}
+                      placeholder="Enter your username"
+                      placeholderTextColor="gray"
+                      value={value}
+                    />
+                    {errors.username && (
+                      <WarningCircle color="#dc2626" size={20} weight="bold" />
+                    )}
+                  </View>
+                )}
+              />
+              {errors.username && (
+                <Text className="pbk-b3 mb-4 text-red-600">
+                  {errors.username.message}
+                </Text>
+              )}
+              <View className="mb-5">
+                <Text className="pbk-b3 text-gray-600">
+                  This is your display name. You can change it later.
+                </Text>
+              </View>
+
+              <Text className="pbk-b2 mb-1.5 text-base-white">Name</Text>
+              <Controller
+                control={control}
+                name="name"
+                render={({ field: { onChange, value } }) => (
+                  <View
+                    className={`mb-4 min-h-14 flex-row items-center justify-between rounded-xl border ${errors.name ? "border-red-600" : "border-gray-920"} px-3 py-2`}
+                  >
+                    <TextInput
+                      autoCapitalize="words"
+                      className="flex-1 text-base-white placeholder:pbk-b1"
+                      onChangeText={(text) => {
+                        onChange(text);
+                      }}
+                      placeholder="Enter full name"
+                      placeholderTextColor="gray"
+                      value={value}
+                    />
+                    {errors.name && (
+                      <WarningCircle color="#dc2626" size={20} weight="bold" />
+                    )}
+                  </View>
+                )}
+              />
+              {errors.name && (
+                <Text className="pbk-b3 mb-4 text-red-600">
+                  {errors.name.message}
+                </Text>
+              )}
```

Comment:
Again a spacing suggestion that works when we have an error or not. This is similar to the View you had set up for the username subtext.

```suggestion
             <View className="mb-4">
                {errors.name && (
                  <Text className="pbk-b3 text-red-600">
                    {errors.name.message}
                  </Text>
                )}
              </View>
```

Reply:
Honestly this would be nice to get in a common form component so we don't have to keep remembering. I made at ticket for form standardization: https://linear.app/rookies-fantasy/issue/ENG-14/standardize-forms

My only actual comment on this specifically is to double check that the spacing is consistent between the fields whatever we decide to do